examples/complete/versions.tf

```
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.0.0"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 2.0"
+      version = ">= 4.22.0"
```

What was changed added after 4.9.0 that requires a later version?
```suggestion
      version = ">= 4.9.0"
```